src/sage/structure/coerce_actions.pyx

```diff
@@ -781,7 +781,7 @@ cdef class IntegerMulAction(IntegerAction):
         Check that large multiplications can be interrupted::
 
             sage: # needs sage.schemes
-            sage: alarm(0.001); 2^(10^7) * P
+            sage: alarm(0.001); 2^(10^7) * P # known bug: macos, random failures, see #35973
```

Comment:
As I explained in https://github.com/sagemath/sage/pull/36372#issuecomment-1868342056 (see also https://github.com/sagemath/sage/pull/36372#issuecomment-1868386854, https://github.com/sagemath/sage/pull/36372#issuecomment-1742417444), explanations should go into parentheses because this is what our parsing/unparsing code knows to handle.

Reply:
Done